packages/lesswrong/components/recentDiscussion/RecentDiscussionSubscribeReminder.tsx

```diff
@@ -160,57 +176,63 @@ const RecentDiscussionSubscribeReminder = ({classes}: {
       </AnalyticsInViewTracker>
     </AnalyticsContext>
   }
-  
+
   if (loading) {
-    return <div>
+    return <div className={classes.root}>
       <Loading/>
     </div>
   } else if (subscriptionConfirmed) {
     return <AnalyticsWrapper branch="already-subscribed">
-      You are subscribed to the best posts of LessWrong!
+      <div className={classes.message}>
+        <CheckRounded className={classes.checkIcon} />
+        You are subscribed to the best posts of {forumType}!
+      </div>
```

Comment:
I added some icons to make my eyes glaze over this less

Reply:
Can you change this to say "You're subscribed to the EA Forum Digest!"

I think it's good to stick with standard terminology, i.e. call it the digest every time we reference it. (I understand that this was from LW, not something you wrote.)